src/models/kernels/reservoir/Reservoir.cpp

```diff
@@ -143,14 +144,11 @@ double Reservoir::response_meters_per_second(double in_flux_meters_per_second, i
     }
 
     //If storage is greater than maximum storage, set to maximum storage and return excess water.
-    if (state.current_storage_height_meters > parameters.maximum_storage_meters)
-    {
+    if (state.current_storage_height_meters > parameters.maximum_storage_meters) {
         /// \todo TODO: Return appropriate warning
         cout << "WARNING: Reservoir calculated a storage above the maximum storage."  << endl;
-
+        excess_water_meters = state.current_storage_height_meters - parameters.maximum_storage_meters;
```

Comment:
@jdmattern-noaa 😄 The little things!  Good catch @robertbartel

Reply:
Yes, my mistake here. Thanks @robertbartel!